docs/docs/building-and-running.md

```diff
@@ -60,14 +60,9 @@ You can also run the same command in an individual component's directory (i.e. `
 
 Helm is the package manager for Kubernetes - we use it to make it so that you can install all the various services you need for MAGDA at once. To install, follow the instructions at [https://github.com/helm/helm#install](https://github.com/helm/helm#install).
 
-In a nutshell, once you have helm installed, this is how you initialise helm and Tiller.
+Magda uses [Helm 3](https://github.com/helm/helm/releases/tag/v3.1.1).
```

Comment:
Actually, line 20 above mentioned helm2 requirement.
You probably  want to change it to helm 3 with link: https://helm.sh/docs/intro/install/
If you do so, you probably don't need to repeat helm 3 requirement here.

Reply:
Done